### PR TITLE
Update proto files

### PR DESCRIPTION
**Pull request by ToniRamirezM**
### What does this PR do?

- Updates proto files for executor and prover.
- Enables tx_hash test for executor.

### Reviewers

Main reviewers:

@arnaubennassar 
@fgimenez 
@tclemos

### CODE REVIEW

**Review by fgimenez — APPROVED** [review]

**Review by arnaubennassar — APPROVED** [review]

File: proto/src/proto/zkprover/v1/zk_prover.proto [hunk 1]
```diff
@@ -13,7 +13,7 @@ message Version {
 /**
  * Define all methods implementes by the gRPC
  * GetStatus: get server report about its current state (non-blocking call)
- * GenProof: ask prover to start proof generation. If prover is busy, request is queued (non-blocking call)
+ * GenProof: ask prover to start proof generation. If prover is biusy, request is queued (non-blocking call)
```
Comment (tclemos) [inline proto/src/proto/zkprover/v1/zk_prover.proto:16]: Typo

File: proto/src/proto/zkprover/v1/zk_prover.proto [hunk 2]
```diff
@@ -45,7 +45,7 @@ message GenProofRequest {
  * @dev CancelRequest
  * @param {id} - proof identifier
  */
-message CancelRequest {
+ message CancelRequest {
```
Comment (tclemos) [inline proto/src/proto/zkprover/v1/zk_prover.proto:48]: Not needed

File: proto/src/proto/zkprover/v1/zk_prover.proto [hunk 3]
```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package zkprover.v1;
 
-option go_package = "./pb";
+option go_package = "github.com/0xPolygonHermez/zkevm-node/proverclient/pb";
```
Comment (tclemos) [inline proto/src/proto/zkprover/v1/zk_prover.proto:5]: Is this a definitive change?

**Review by tclemos — APPROVED** [review]
Just small fixes that will not block this PR to be merged but would be good to get fixed before merging

**Pull request merged**